ios/FluentUI/Command Bar/CommandBarCommandGroupsView.swift

```diff
@@ -77,15 +77,28 @@ class CommandBarCommandGroupsView: UIView {
     private func updateButtonGroupViews() {
         updateItemsToButtonsMap()
         buttonGroupViews = itemGroups.map { items in
-                CommandBarButtonGroupView(buttons: items.compactMap { item in
-                    guard let button = itemsToButtonsMap[item] else {
-                        preconditionFailure("Button is not initialized in map")
-                    }
-                    item.propertyChangedUpdateBlock = { _ in
+            let buttons: [CommandBarButton] = items.compactMap { item in
+                guard let button = itemsToButtonsMap[item] else {
+                    preconditionFailure("Button is not initialized in map")
+                }
+                return button
+            }
+
+            let group = CommandBarButtonGroupView(buttons: buttons)
+
+            for item in items {
+                if let button = itemsToButtonsMap[item] {
+                    item.propertyChangedUpdateBlock = { _, updateGroupViewState in
```

Comment:
"updateGroupViewState" : i am also bad at naming things but can we think how this can be more clear swift boolean naming?

Reply:
`shouldUpdateGroupState` seems a bit better to me. The `should` prefix hints at the fact that it is a boolean.